src/components/Community/Community.js

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Moment from 'react-moment';
+import Header from '../Header/Header';
+import Spinner from '../Spinner/Spinner';
+import Footer from '../Footer/Footer';
+import Sidebar from '../Sidebar/Sidebar';
+import Input from '../Inputs/Input';
+import Button from '../Button';
+import * as actions from '../../../store/actions/community';
+import './community.scss';
+import sideLogo from '../../assets/images/logo.png';
+
+export class Community extends PureComponent {
+  state = {
+    messages: [],
+  };
+
+  componentDidMount() {
+    const getCommMessages = async () => {
+      const { getCommunityMessages } = this.props;
+      await getCommunityMessages();
+      const { message } = this.props;
+      this.setState({ messages: message });
+    };
+    getCommMessages();
+  }
+
+  render() {
+    const spinner = (
+      <div className="spinner-position">
+        <Spinner />
+      </div>
+    );
+
+    const { messages } = this.state;
+    const { loading } = this.props;
+    return (
+      <div>
+        <Header />
+        <div className="main_com">
+          <Sidebar>
+            <div className="d-aside">
+              <div className="c-side">
+                <span className="c-logo">
+                  <img src={sideLogo} alt="logo" />
+                  Community
+                </span>
+              </div>
+              <ul className="d-links">
+                <li className="module-active">
+                  <Link to="/community">Forum</Link>
+                </li>
+                <li>
+                  <Link to="/profile">Profile</Link>
+                </li>
+                <li>
+                  <Link to="/payment">Messages</Link>
+                </li>
+                <li>
+                  <Link to="/aboutus">Help</Link>
+                </li>
+              </ul>
+            </div>
+          </Sidebar>
+          <div className="main_sec">
+            <div className="input_section">
+              <Input id="inp" placeholder="Title" />
+              <textarea rows="3" placeholder="enter your message..." />
+              <Button text="Post message" classname="com_btn" />
+              <hr />
+            </div>
+            <div className="posts">
+              {loading ? (
+                spinner
+              ) : (
+                messages.map(msg => (
+                  <div className="posts_section" key={typeof msg.id === 'number' ? msg.id : ''}>
+                    <h4 className="post_title">
+                      {msg.owner !== undefined ? msg.owner.lastName : ''}
```

Comment:
This can be made more succinct and cleaner by doing this 👇 
{msg.owner && msg.owner.lastName}